frontend/pages/_document.jsx

```diff
@@ -87,4 +91,4 @@ class MyDocument extends Document {
   }
 }
 
-export default MyDocument;
\ No newline at end of file
+export default MyDocument;
```

Comment:
Since we [changed `MyApp` to `App`](https://github.com/ModularHistory/modularhistory/pull/289/files#diff-9c9ff648160c4ca56ad3e0809b4c493a3e9282c6546de0e4f7cf797ed8022560R7), should we change `MyDocument` to `Document`?

Reply:
Yup.